llvm/test/CodeGen/RISCV/rvv/fixed-vectors-extract-i1.ll

```diff
@@ -221,7 +221,7 @@ define i1 @extractelt_v64i1(ptr %x, i64 %idx) nounwind {
 ; RV64-NEXT:    vsetvli zero, a2, e8, m4, ta, ma
 ; RV64-NEXT:    vle8.v v8, (a0)
 ; RV64-NEXT:    vmseq.vi v12, v8, 0
-; RV64-NEXT:    vsetivli zero, 1, e64, m1, ta, ma
+; RV64-NEXT:    vsetvli zero, a2, e64, m1, ta, ma
```

Comment:
We're preserving the AVL here but changing the VTYPE, which is in theory the minimal change needed to get the correct state. This looks like a case where the postpass could be replacing it with 1 to reduce the live range